packages/fxa-auth-server/lib/metrics/glean/index.ts

```diff
@@ -93,7 +99,8 @@ const createEventFn =
 
       const metrics = {
         user_agent: request.headers['user-agent'],
-        ip_address: request.app.clientAddress,
+        ip_address:
+          options?.skipClientIp === true ? '' : request.app.clientAddress,
```

Comment:
@akkomar is an empty string going to work here?

Reply:
Yes.